src/Host/Client/Impl/Session/RSessionProvider.cs

```diff
@@ -35,9 +39,10 @@ public class RSessionProvider : IRSessionProvider {
         public event EventHandler BrokerChanged;
         public event EventHandler<BrokerStateChangedEventArgs> BrokerStateChanged;
 
-        public RSessionProvider(IRSessionProviderCallback callback = null) {
+        public RSessionProvider(IActionLog log, IRSessionProviderCallback callback = null) {
```

Comment:
That will minimize amount of changes

---
In reply to: [80987120](https://github.com/Microsoft/RTVS/pull/2408#discussion_r80987120) [](ancestors = 80987120)